Makefile

```diff
@@ -5,7 +5,7 @@ JAIL_NET?=16
 MYPYPATH = $(shell pwd)/.travis/mypy-stubs
 
 PYTHON_VERSION ?= $(TRAVIS_PYTHON_VERSION)
-SELECTED_PYTHON_VERSION != if [ "$(PYTHON_VERSION)" != "" ]; then echo $(PYTHON_VERSION); else pkg info -g 'python3*' | cut -d'-' -f1 | sed 's/^python//' | sort -n | tail -n1 | sed -r 's/^([0-9])([0-9]+)/\1.\2/'; fi
+SELECTED_PYTHON_VERSION != if [ "$(PYTHON_VERSION)" != "" ]; then echo $(PYTHON_VERSION); else pkg query '%dn' 'python3' | sort -un | sed -r 's/^python//;s/^([0-9])([0-9]+)/\1.\2/' | tail -n1 ; fi
```

Comment:
Can you explain the expected output of `pkg query '%dn' 'python3'` please?

Here is some output from a system with python37 installed:
```console
root@freebsd-debug:~ # pkg query '%dn' 'python3'
root@freebsd-debug:~ # echo $?
69
root@freebsd-debug:~ # pkg query '%dn' 'python37'
readline
libffi
gettext-runtime
root@freebsd-debug:~ # echo $?
0
```

Reply:
we explicitly `pkg install -y python3`
this pulls in the system-default python3, which should be `python37` as of 12.1-RELEASE.
*then* we check `pkg query '%dn' 'python3'` and that gives us `python37`.